third_party/remoteapis/build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -2073,4 +2073,9 @@ message RequestMetadata {
   // There is no expectation that this value will have any particular structure,
   // or equality across invocations, though some client tools may offer these guarantees.
   string configuration_id = 7;
+
+  // If Bazel automatically retries an invocation, this value is set to the
+  // number of invocation attempts that have been started so far. For example, a
+  // value of 2 would indicate that it is the second attempt.
```

Comment:
What are the semantics of a 0 value?

Reply:
That is invalid, and is checked against in https://github.com/bazelbuild/bazel/commit/2ab3be22e5d3605673fea6d43783c057be46a2b9#diff-a1596dc84d5ee9147cf41cac44f7fa747166950a0f8ad3f711d8fcd3c36c0900R197

`checkArgument(attemptNumber >= 1);`